docs-en/chapter_array_and_linkedlist/list.md

```diff
@@ -254,9 +254,9 @@ Lists are essentially arrays, so accessing and updating elements can be done in
     nums.items[1] = 0; // Update the element at index 1 to 0  
     ```
 
-### Inserting and Deleting Elements
+### Inserting and Removing Elements
 
-Compared to arrays, lists can freely add and remove elements. Adding elements at the end of the list has a time complexity of $O(1)$, but the efficiency of inserting and deleting elements is still the same as in arrays, with a time complexity of $O(n)$.
+Compared to arrays, lists offer more flexibility in adding and removing elements. While adding elements to the end of a list is an $O(1)$ operation, the time complexity for inserting and removing elements elsewhere in the list remains the same as in arrays, which is $O(n)$.
```

Comment:
optional you could do:
"the time complexity for inserting and removing elements elsewhere in the list remains the same as in arrays, which is $O(n)$." -> "the efficiency of inserting and removing elements elsewhere in the list remains the same as in arrays, with a time complexity of $O(n)$." 
to be a bit more explicit.